.github/workflows/test-and-publish-nightly.yml

```diff
@@ -63,7 +63,8 @@ jobs:
               make COMMIT_TAG=${GITHUB_SHA:0:7} CC=/usr/local/musl/bin/musl-gcc release coverage -j
             ;;
           esac
-          echo "archive=$(basename $(find . -type f -iname "async-profiler-*" ))" >> $GITHUB_OUTPUT
+          echo "archive=$(find . -type f -name "async-profiler-*" -not -name "*-sym*" -exec basename {} \; || echo "")" >> $GITHUB_OUTPUT
+          echo "sym_archive=$(find . -type f -name "async-profiler-*-sym*" -exec basename {} \; | awk '{print; exit} END{if(NR==0) print "IGNORE"}')" >> $GITHUB_OUTPUT
```

Comment:
> No files were found with the provided path: IGNORE.

Isn't there a less tricky way to skip macOS?

Reply:
Github actions step did not accept empty string as a valid value.

Reply:
Will skip with an "is macos" check.